README.md

```diff
@@ -45,7 +43,7 @@ The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool
 
     ```bash
     docker pull ghcr.io/pangenome/pggb:latest
-    nextflow run ./main.nf -profile docker --input_fasta ~/git/test-datasets/data/DRB1-3123/DRB1-3123.fa.gz
+    nextflow run nf-core/pangenome -profile docker --input_fasta ~/git/test-datasets/data/DRB1-3123/DRB1-3123.fa.gz
```

Comment:
`~/git/test-datasets/data/DRB1-3123/DRB1-3123.fa.gz`

This is your local path, right?

Reply:
Shall I point to the test data again or to an arbitrary path?

Reply:
Yes, you are right.

Reply:
Test data again, please.

Reply:
According to nf-core standards you just write something like

`--input_fasta myfasta.fa.gz`

Note that we should use --input and not custom input parameters

Reply:
Done.